rllib/tests/test_pettingzoo_env.py

```diff
@@ -0,0 +1,40 @@
+"""Integration test: (1) pendulum works, (2) single-agent multi-agent works."""
```

Comment:
Can you add an entry to rllib/BUILD so that this test is run by travis automatically?